sdk/state/open_test.go

```diff
@@ -1,14 +1,131 @@
 package state
 
 import (
+	"strconv"
 	"testing"
 	"time"
 
 	"github.com/stellar/go/keypair"
 	"github.com/stellar/go/network"
+	"github.com/stellar/go/xdr"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func TestOpenAgreement_Equal(t *testing.T) {
```

Comment:
💭 `cmp` compares signatures as well? In which case can we add a test case of 2 different signatures to

Reply:
Yup. The third last and last test case test that actually 😄.

Reply:
ah I meant two existing sigs that are different, not existent and non-existent

I don't see that case being tested 🤔

Reply:
Oh gotcha. That's what the last test case was supposed to be but looks like it has a bug. I'll fix it in another PR.